app/config/phansible/syspackages.yml

```diff
@@ -7,3 +7,4 @@ syspackages:
   - imagemagick
   - nodejs
   - npm
+
```

Comment:
Please don't add changes that are not part of the PR
Or changes that don't add anything.